### PR TITLE
Phish.AI - Deprecated. Partner has declared end of life

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/36053

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/PhishAI/Playbooks/playbook-Detonate_URL_-_PhishAI.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 id: Detonate URL - Phish.AI
 version: -1
+hidden: true
 name: Detonate URL - Phish.AI
 description: Detonates a URL using the Phish.AI integration.
```
Comment (yaakovi) [inline Packs/PhishAI/Playbooks/playbook-Detonate_URL_-_PhishAI.yml:5]: update description too

**Review by yaakovi — APPROVED** [review]

**Review by amshamah419 — APPROVED** [review]
Looks good

**Comment by ShahafBenYakir** [comment]
@bakatzir Why not deprecating the pack?

**Comment by bakatzir** [comment]
@ShahafBenYakir I also added hidden tp the metadata
https://github.com/demisto/content/blob/bb57f4a9c4d2298f7296b65df24bd95ca98352d1/Packs/PhishAI/pack_metadata.json

**Pull request merged**